Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js

```diff
@@ -45,19 +45,6 @@ type Props = $ReadOnly<{|
    */
   keyboardDismissMode?: ?('none' | 'on-drag'),
 
-  /**
-   * Specifies the background color of the drawer. The default value is white.
-   * If you want to set the opacity of the drawer, use rgba. Example:
-   *
-   * ```
-   * return (
-   *   <DrawerLayoutAndroid drawerBackgroundColor="rgba(0,0,0,0.5)">
-   *   </DrawerLayoutAndroid>
-   * );
-   * ```
-   */
-  drawerBackgroundColor: ColorValue,
```

Comment:
@lunaleaps i think we can remove this because, when a user passes a custom prop to the component it will pick it up and replace the default value which is currently - "white". if we keep it here it would raise an error saying that drawerBackgroundColor is present in "Props"(read only flow interface) and not present inside the props in the DrawerLayoutAndroid Class component. 

When the "static props" field was present this wasn't an issue as the class by default added it to the props object.